addon/components/horizontal-bar-chart.js

```diff
@@ -237,13 +237,24 @@ const HorizontalBarChartComponent = ChartComponent.extend(FloatingTooltipMixin,
     };
   }),
 
+  /*
+   * Ensures that grouping labels will be on the left when all data is negative or 0
```

Comment:
This comment describes what this function is used for, but not what it does. It should do that as well.